acceptance_test.go

```diff
@@ -64,26 +62,40 @@ func (d *driver) GenerateRecord(_ *testing.T, operation opencdc.Operation) openc
 	}
 }
 
+func (d *driver) WriteTimeout() time.Duration {
+	return time.Minute
+}
+
 func TestAcceptance(t *testing.T) {
 	dsn := os.Getenv(envNameDSN)
 	if dsn == "" {
 		t.Skipf("%s env var must be set", envNameDSN)
 	}
 
-	cfg := map[string]string{
+	table := fmt.Sprintf("conduit_test_%d", time.Now().UnixNano())
+
+	srcCfg := map[string]string{
 		srcConfig.ConfigDsn:            dsn,
-		srcConfig.ConfigTable:          fmt.Sprintf("conduit_test_%d", time.Now().UnixNano()),
+		srcConfig.ConfigTable:          table,
 		srcConfig.ConfigOrderingColumn: "col1",
 	}
 
+	destCfg := map[string]string{
+		destConfig.ConfigDsn:        dsn,
+		destConfig.ConfigTable:      table,
+		destConfig.ConfigKeyColumns: "col1",
+	}
+
 	sdk.AcceptanceTest(t, &driver{
 		ConfigurableAcceptanceTestDriver: sdk.ConfigurableAcceptanceTestDriver{
 			Config: sdk.ConfigurableAcceptanceTestDriverConfig{
 				Connector:         Connector,
-				SourceConfig:      cfg,
-				DestinationConfig: cfg,
-				BeforeTest:        beforeTest(cfg),
-				AfterTest:         afterTest(cfg),
+				SourceConfig:      srcCfg,
+				DestinationConfig: destCfg,
+				BeforeTest:        beforeTest(destCfg),
+				AfterTest:         afterTest(destCfg),
+				// some parameters contains "*" which is a valid character in source parameter
```

Comment:
What exactly is the issue here? Maybe we need to adjust the acceptance tests in the SDK to handle this.

Reply:
We have a regex which does not allow "*" in parameter name, but we have parameters like [this](https://github.com/ConduitIO/conduit-connector-generator/blob/main/config_paramgen.go)

Reply:
We'll fix that here: https://github.com/ConduitIO/conduit-connector-sdk/pull/213.